test/test_torch.py

```diff
@@ -1505,156 +1517,102 @@ def test_generator_cpu(self):
-            engine.fast_forward(4)
-            sample = engine.draw(n=4)
-            torch.testing.assert_allclose(sample, ref_sample[4:])
-            # alternate fast forwarding with sampling
-            engine.reset()
-            even_draws = []
-            for i in range(8):
-                if i % 2 == 0:
-                    even_draws.append(engine.draw())
-                else:
-                    engine.fast_forward(1)
-            torch.testing.assert_allclose(
-                ref_sample[[i for i in range(8) if i % 2 == 0]],
-                np.concatenate(even_draws),
-            )
-
-        def test_sobolengine_fast_forward_scrambled(self):
-            self.test_sobolengine_fast_forward(scramble=True)
-
-        def test_sobolengine_distribution(self, scramble=False):
-            d = 50
-            engine = torch.quasirandom.SobolEngine(d, scramble=scramble, seed=123456)
-            sample = engine.draw(1024)
-            torch.testing.assert_allclose(
-                torch.mean(sample, dim=0), torch.full((d,), 0.5), atol=2, rtol=2
-            )
-            torch.testing.assert_allclose(
-                np.percentile(sample, 25, axis=0), np.repeat(0.25, d), atol=2, rtol=2
-            )
-            torch.testing.assert_allclose(
-                np.percentile(sample, 75, axis=0), np.repeat(0.75, d), atol=2, rtol=2
-            )
-
-        def test_sobolengine_distribution_scrambled(self):
-            self.test_sobolengine_distribution(scramble=True)
-
-        def test_sobolengine_draw_base2(self, scramble=False):
-            ref_sample = self._sobol_reference_samples(scramble=scramble)
-            engine = torch.quasirandom.SobolEngine(2, scramble=scramble, seed=123456)
-            sample = engine.draw_base2(2)
-            self.assertEqual(ref_sample[:4], sample)
-            # resampling still having N=2**n
-            sample = engine.draw_base2(2)
-            self.assertEqual(ref_sample[4:8], sample)
-
-        def test_sobolengine_draw_base2_scrambled(self):
-            self.test_sobolengine_draw_base2(scramble=True)
-
-        def test_sobolengine_raise(self):
-            maxdim = torch.quasirandom.SobolEngine.MAXDIM
-            with self.assertRaises(ValueError):
-                torch.quasirandom.SobolEngine(maxdim + 1)
-
-        def test_sobolengine_high_dim(self):
-            engine = torch.quasirandom.SobolEngine(1111, scramble=False, seed=123456)
-            samples1 = engine.draw()
-            vals1, counts1 = torch.unique(samples1, return_counts=True)
-            samples2 = engine.draw()
-            vals2, counts2 = torch.unique(samples2, return_counts=True)
-            self.assertEqual(vals1.item(), 0.0)
-            self.assertEqual(counts1.item(), 1111)
-            self.assertEqual(vals2.item(), 0.5)
-            self.assertEqual(counts1.item(), 1111)
+        def test_sobolengine_unscrambled_lowdim(self):
```

Comment:
It seems like there are a lot of unrelated changes in this PR? Maybe a bad merge?

Reply:
I guess i was having some trouble during the rebase, probably a bad merge.. Will rebasing it again do the trick?

Reply:
It might be simpler copy your changes, reset the file to the latest commit, then paste your changes in.